hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -817,7 +817,7 @@ public String pathToKey(Path path) {
     if (OzoneFSUtils.isValidName(key)) {
       key = path.toUri().getPath();
     } else {
```

Comment:
Not introduced in this patch, but would be nice to remove duplicate conversion:

```suggestion
    if (!OzoneFSUtils.isValidName(key)) {
```